fuel/downloaders/base.py

```diff
@@ -117,6 +117,13 @@ def default_downloader(directory, urls, filenames, url_prefix=None,
                 os.remove(f)
     else:
         print('Downloading ' + ', '.join(filenames) + '\n')
+        try:
+            os.makedirs(directory)
+        except OSError as e:
```

Comment:
Another (IMHO) more straightforward way of implementing this would be

``` python
if not os.path.exists(directory):
    os.makedirs(directory)
```

What do you think?